src/site/stages/build/drupal/process-lovell-pages.js

```diff
@@ -243,49 +268,150 @@ function updateLovellSwitchLinks(page, pages) {
   return page;
 }
 
+/**
+ * For each listing page in tricareOrVaPages, finds the first occurrence of that listing page type in
+ * federalPages and injects pastEvents and reverseFieldListingNode from that entity into
+ * corresponding properties on the listing page from tricareOrVaPages.
+ *
+ * @param {*} tricareOrVaPages Listing pages that will have pastEvents and reverseFieldListingNode properties merged from federalPages
+ * @param {*} federalPages Listing pages to merge into tricareOrVaPages
+ * @returns
+ */
+function combineLovellListingPages(tricareOrVaPages, federalPages) {
+  return tricareOrVaPages.map(listingPage => {
+    let pastObjectLabel;
+    switch (listingPage.entityBundle) {
+      case 'press_release_listing':
+        pastObjectLabel = 'pastPressReleases';
+        break;
+      case 'story_listing':
+        pastObjectLabel = 'pastNewsStories';
+        break;
+      default:
+        pastObjectLabel = 'pastEvents';
+        break;
+    }
```

Comment:
Did we need a catchall (default) for event?  If we don't really need the catchall it might be tighter to use the actual value as a map to avoid something unexpected
```suggestion
    const typePastMap = [
      'event_listing' => 'pastEvents',
      'press_releases_listing' => 'pastPressReleases',
      'story_listing' => 'pastNewsStories',
    ];
    pastObjectLabel = typePastMap[listingPage.entityBundle];   
```

Reply:
This looks like a good idea, going to try it out now and commit it if it works.